addons/AsepriteWizard/animated_sprite/import_plugin.gd

```diff
@@ -58,7 +63,7 @@ func get_import_options(i):
 	]
 
 
-func get_option_visibility(option, options):
+func _get_option_visibility(option, options):
```

Comment:
This is missing the `path` parameter -> `func _get_option_visibility(path, option, options):`

Reply:
Thanks @aaaaaaaaargh! I changed that. Your comments are highly welcome. One thing that is still not working are the error messages, if you have a fix here, I'd love to integrate it.

Reply:
@floriandotorg Sorry, I was overlooking your comment as well -_- 

I did have a patch but I must have forgotten to create a PR and it seems to be lost now. Sorry about that, but if you need any further assistance here I'd be happy to help.